### ISSUE TITLE
loss=nan

### ISSUE THREAD

**Issue by MauricioGS99** [takuseno/d3rlpy#255]
Hello,
I'm trying to run offline RL where the state is formed by 75 or 100 variables (sampled from a bayesian network). The collected samples are in a data frame called "data", and I run the following.

```actions=data['treatment']

observations_dwh=data[['disease','weight','heartattack']].to_numpy()

rewards = data['variable74']

m=len(actions)

terminals = np.repeat(1,m)

dataset_dwh = MDPDataset(observations_dwh, actions, rewards, terminals)

train_episodes_dwh, test_episodes_dwh = train_test_split(dataset_dwh)

q_func_dwh=d3rlpy.algos.DQN()

q_func_dwh.fit(train_episodes_dwh,test_episodes_dwh,scorers={'advantage': discounted_sum_of_advantage_scorer,
                                              'td_error': td_error_scorer, # smaller is better
                                              'value_scale': average_value_estimation_scorer
                                             })`


And it runs quite good, except that the loss=nan from the first step,
any idea why?

Thanks.

**Comment by MauricioGS99** [comment]
Thanks for your reply! 
No, in fact the actions are only 0 and 1's.
But I noticed that my sampling scheme (not shown in this code) was producing arbitrarily high numbers (but finite, not infs). Once I fixed that, d3rlpy went back to normal.

**Comment by takuseno** [comment]
Sorry for the late response, but happy to hear it worked out! Feel free to re-open this issue when you get the same issue again.
